apps/meteor/ee/app/livechat-enterprise/server/api/lib/tags.js

```diff
@@ -1,5 +1,5 @@
 import { escapeRegExp } from '@rocket.chat/string-helpers';
-import LivechatTag from '@rocket.chat/models';
+import { LivechatTag } from '@rocket.chat/models';
```

Comment:
What do you think about convert to TS ? this way this is never going to happen again

Reply:
Yeah sounds good! Converted it to typescript. Please check